pipeline/dbt/models/intermediate/sources/dora/int_dora__services.sql

```diff
@@ -48,7 +48,10 @@ final AS (
         services.modes_accueil                           AS "modes_accueil",
         services.modes_orientation_accompagnateur        AS "modes_orientation_accompagnateur",
         services.modes_orientation_accompagnateur_autres AS "modes_orientation_accompagnateur_autres",
-        services.modes_orientation_beneficiaire          AS "modes_orientation_beneficiaire",
+        CASE
```

Comment:
https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/

je serais content qu'on explique ici avec un commentaire pourquoi on a décidé de mettre NULL si un seul des modes d'orientation est "pro" ? 
Pourquoi on a pas décidé d'ignorer tout le service carrément, ou de retirer ce mode si d'autres sont dispo ?

Bref, un commentaire ici permettra aux futurs dev de comprendre le pourquoi de ce code si jamais ils veulent etre en capacité de le modifier.

Ne doit-on pas demander à Dora pourquoi ils nous envoient cette valeur au fond, dans leur API ? Ils pourraient pas filtrer eux memes ?